.rhcicd/clowdapp-aggregator.yaml

```diff
@@ -61,8 +61,8 @@ objects:
           value: ${ENV_NAME}
         - name: PROMETHEUS_PUSHGATEWAY_URL
           value: ${PROMETHEUS_PUSHGATEWAY}
-        - name: NOTIFICATIONS_AGGREGATOR_ORG_PREF_ENABLED
```

Comment:
Did you mean to remove this old feature in this PR? It looks like the branch/PR contains two unrelated topics.